test/preconditioner/batch_ilu_isai_kernels.cpp

```diff
@@ -66,26 +66,35 @@ class BatchIluIsai : public CommonTestFixture{
 
     BatchIluIsai()
         : 
-          mtx(gko::share(gko::test::generate_uniform_batch_random_matrix<Mtx>(
-              nbatch, nrows, nrows,
-              std::uniform_int_distribution<>(min_nnz_row, nrows),
+          mtx_small(gko::share(gko::test::generate_uniform_batch_random_matrix<Mtx>(
+              nbatch, nrows_small, nrows_small,
+              std::uniform_int_distribution<>(min_nnz_row_small, nrows_small),
+              std::normal_distribution<real_type>(0.0, 1.0), rand_engine, true,
+              ref))),
+          mtx_big(gko::share(gko::test::generate_uniform_batch_random_matrix<Mtx>(
+              nbatch, nrows_big, nrows_big,
+              std::uniform_int_distribution<>(min_nnz_row_big, nrows_big),
               std::normal_distribution<real_type>(0.0, 1.0), rand_engine, true,
               ref)))
     {}
 
     std::ranlux48 rand_engine;
 
     const size_t nbatch = 9;
-    const index_type nrows = 15;
-    const int min_nnz_row = 3;
-    std::shared_ptr<const Mtx> mtx;
+    const index_type nrows_small = 15;
+    const int min_nnz_row_small = 3;
+    std::shared_ptr<const Mtx> mtx_small;
+    const index_type nrows_big = 50;
+    const int min_nnz_row_big = 33;
```

Comment:
Can we have one matrix with some rows having nnz_per_row < 32 and some having nnz_per_row > 32 ?

Reply:
Should I just make `min_nnz_row_big = 28` or something like that? That way there is a possibility that some rows have nnz_per_row <= 32 while some have nnz_per_row > 32.

Reply:
Yes, that should be okay.